### PR TITLE
Dotnet support

### PR DESCRIPTION
**Pull request by doppleware**

### CODE REVIEW

File: src/languageSupport.ts [hunk 1]
```diff
@@ -18,6 +18,71 @@ export interface ISupportedLanguage
...
+    public extractSymbolInfos(document: vscode.TextDocument, docSymbols: DocumentSymbol[]): SymbolInfo[] {
+
+        const symbolInfos = this.extractFunctions(document.uri.toModulePath(), '', docSymbols);
+        return symbolInfos;
+    }
+
+    private removeParenthesis(funcName :string):string{
+        var parenthesisIndex = funcName.indexOf('(');
+        if (parenthesisIndex>0){
+            return funcName.substring(0,3);
```
Comment (asafc64) [inline src/languageSupport.ts:44]: What is the case here? Why the first 3 chars?

File: src/views/errorFlow/errorFlowStackView.ts [hunk 2]
```diff
@@ -228,6 +271,21 @@ class ErrorFlowDetailsViewProvider implements vscode.WebviewViewProvider, vscode
             summmary: response.summary
         };
     }
+    private findWorkSpace(frame: ErrorFlowFrame): vscode.Uri | undefined {
```
Comment (asafc64) [inline src/views/errorFlow/errorFlowStackView.ts:274]: Please remove it if not in use

File: src/views/errorFlow/errorFlowStackView.ts [hunk 3]
```diff
@@ -576,14 +670,37 @@ class ErrorFlowDetailsViewProvider implements vscode.WebviewViewProvider, vscode
...
-        const moduleRootFolder = moduleName.split('/').firstOrDefault();
-        const moduleWorkspace = vscode.workspace.workspaceFolders?.find(w => w.name == moduleRootFolder);
-        const workspaceUri = moduleWorkspace
-            ? vscode.Uri.joinPath(moduleWorkspace.uri, '..', moduleName)
-            : undefined;
-        return workspaceUri;
+
+    private async lookupCodeObjectByFullName(name:string) : Promise<vscode.SymbolInformation[]>{
+
+       return await vscode.commands.executeCommand("vscode.executeWorkspaceSymbolProvider", name);
```
Comment (asafc64) [inline src/views/errorFlow/errorFlowStackView.ts:676]: Cool!

File: src/languageSupport.ts (hunk 1 above)
Comment (doppleware) [reply src/languageSupport.ts:44]: its a bug! fixed now, its to remove the ()

**Pull request merged**